### ISSUE TITLE
Group kfold

### ISSUE THREAD

**Issue by K-Mike** [trent-b/iterative-stratification#10]
Do you plan to do an analog of 
https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GroupKFold.html

**Comment by trent-b** [comment]
Thank you for the idea. I have not planned to do an analog of GroupKFold. It is unclear to me how one would handle the competing objectives of separating groups by fold and stratifying multiple labels across folds.
